docs/development/code_guide.rst

```diff
@@ -5,14 +5,73 @@ Code Development Guidelines
 This document describes the coding requirements and guidelines to be
 followed during the development of PlasmaPy and affiliated packages.
 
-Dependencies
-============
+Code written for PlasmaPy must be compatible with Python 3.6 and
+later. Python 2 is not supported by PlasmaPy.
+
+PlasmaPy requires 
+
+* Astropy 2.0 or later
+* NumPy 1.13 or later
+* scipy 0.19 or later
+* matplotlib 2.0 or later
+
+Setting up an environment for development
+=========================================
+
+To obtain PlasmaPy's source code, use the usual
+
+.. code-block:: bash
+
+  git clone https://github.com/PlasmaPy/PlasmaPy.git 
+
+To be able to import PlasmaPy from your source version:
+
+.. code-block:: bash
+
+  cd PlasmaPy
+  python setup.py develop
+
+If you are not working within a virtual environment, this may end in a
+permission error.
+
+To access Git commands on Windows, try `Git Bash <https://git-scm.com/downloads>`_.
+
+Setup procedures for the two most popular virtual
+environments, conda and virtualenv, are listed below.
+
+Conda
+-----
+
+To set up a development environment for PlasmaPy, `the Anaconda
+distribution <https://www.anaconda.com/download/>`_ is strongly
+recommended.
+
+After installing Anaconda, launch any conda environment and enter
+PlasmaPy's repository root directory.
+
+On Windows, the way to do this is via running `Anaconda Prompt` from the
+Start Menu. `Git Bash` may also work if you have added Anaconda to `PATH`.
+
+Afterwards, execute the following:
+
+.. code-block:: bash
+
+    conda env create
```

Comment:
When I run this on the command line in Linux I get:
```
Using Anaconda API: https://api.anaconda.org

SpecNotFound: Can't process without a name
```

Is this command explicitly Windows specific?

Reply:
Nah, it's on Linux. I code badly when under thesis delivery deadline stress, though. It should be fixed now.